core/controllers/feedback_test.py

```diff
@@ -39,6 +42,28 @@
     'text', 'updated_status', 'updated_subject', 'received_via_email']
 
 
+class MockFeedbackAnalyticsAggregator(
+        feedback_jobs_continuous.FeedbackAnalyticsAggregator):
+    """A modified FeedbackAnalyticsAggregator that does not start a new batch
+    job when the previous one has finished.
+    """
+    @classmethod
+    def _get_batch_job_manager_class(cls):
+        return MockFeedbackAnalyticsMRJobManager
+
+    @classmethod
+    def _kickoff_batch_job_after_previous_one_ends(cls):
+        pass
+
+
+class MockFeedbackAnalyticsMRJobManager(
+        feedback_jobs_continuous.FeedbackAnalyticsMRJobManager):
+
+    @classmethod
+    def _get_continuous_computation_class(cls):
+        return MockFeedbackAnalyticsAggregator
```

Comment:
I thought we agreed to avoid doing this? It also seems to be the cause of non-100% code coverage.

Reply:
See this comment: https://github.com/oppia/oppia/pull/6740#discussion_r289951545
@nithusha21 suggested not to create models directly but run continuous jobs instead or use services. I did not find any suitable service to create FeedbackAnalyticsModel so I used the job

Reply:
Sorry, not following the issue here.

If there's no service, I think it's a fair compromise to create it here anyway and file an issue to make a real service later (or better yet, add the service function yourself since you're changing the files anyway).

@nithusha21 WDYT?

Reply:
Hmm I can do that but running the job won't be in the way of achieving 100% coverage. There are a lot of files which run continuous jobs and I also looked in other files and they all create the FeedbackAnalyticsModel though this job

Reply:
I'm still confused about what the concern is, then. Could you please summarize the problem in a comment so @nithusha21 @seanlip and I can give focused feedback in this thread?

(100% coverage isn't the concern here. Tests should flow through the critical path, and this code as is avoids it)

Reply:
Hi @brianrodri I missed this comment earlier. What Rishav has done here follows the pattern set in the tests for feedback continuous jobs. It's been a while since I had worked on these jobs, and I can't seem to remember if there was any reason for writing tests in this fashion. For now, I am happy to leave the tests this way (as it is followed elsewhere). If we come up with a better strategy, we could use that in all locations.

By the way, a service function wouldn't suffice right? The models are populated by the continuous computation job and we need to try to make sure that it has completed generation for the models that already exist. I don't think there should be a service for it. What's your opinion Brian?